src/nvm/file.rs

```diff
@@ -115,6 +115,90 @@ impl FileNvmBuilder {
         self
     }
 
+    #[cfg(target_os = "linux")]
+    fn file_open_with_error_info<P: AsRef<Path>>(
+        &self,
+        do_create: bool, // This denoes whether or not the next argument `options` allows file creating.
+        options: &fs::OpenOptions,
+        filepath: &P,
+    ) -> Result<File> {
+        use std::os::unix::fs::OpenOptionsExt;
+
+        let file_orig = track_io!(options.open(&filepath));
```

Comment:
I feel "open_result" (or something like "XXX_result") is more appropriate name. How do you think?

Reply:
OK, I renamed!